src/transport.cpp

```diff
@@ -920,9 +920,10 @@ namespace quicr {
                       {
                         msg.group_id,
                         msg.object_id,
+                        msg.subscribe_id,
                         msg.payload.size(),
                         msg.priority,
-                        std::nullopt,
+                        msg.priority,
```

Comment:
Priority twice?

Reply:
good catch. It was supposed to still be nullopt.  I've updated that.